articles/troubleshooting/tsconfigtypewarning.md

```diff
@@ -0,0 +1,9 @@
+# The `tsconfig.json` Build Action Warning
+
+You may get an Intellisense warning that reads "One or more tsconfig.json files in this project have their Build Action set to None and as a result may not be correctly used for configuration." as shown below.
```

Comment:
"as shown below" suggests that there should be a screenshot or something.

Reply:
I'm also fine tweaking the wording and leaving out the screenshot.

Reply:
Wording tweaked.